static/src/javascripts/projects/commercial/modules/dfp/Advert.ts

```diff
@@ -14,55 +14,29 @@ type Timings = {
 	lazyWaitComplete: number | null;
 };
 
-const stringToTuple = (size: string): AdSizeTuple => {
-	const dimensions = size.split(',', 2).map(Number);
-
-	// Return an outOfPage tuple if the string is not `{number},{number}`
-	if (dimensions.length !== 2 || dimensions.some((n) => isNaN(n)))
-		return [0, 0]; // adSizes.outOfPage
-
-	return [dimensions[0], dimensions[1]];
+const isSlotName = (slotName: string): slotName is SlotName => {
+	return !!(slotName in slotSizeMappings);
```

Comment:
Do we need the `!!` here?

Reply:
At one point typescript was complaining but you're right🤔